cmd/root_test.go

```diff
@@ -1210,8 +1210,8 @@ func TestQuitQuitQuit(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to dial endpoint: %v", err)
 	}
-	if resp.StatusCode != http.StatusBadRequest {
-		t.Fatalf("expected a 400 status, got = %v", resp.StatusCode)
+	if resp.StatusCode != http.StatusOK {
```

Comment:
Could you add unit test coverage for a method that's not a GET or POST as well?

Reply:
sure!

Reply:
fixed test aligned with cloud-sql-proxy

Reply:
Thanks!